sdk/cosmosdb/cosmos/src/request/DedicatedGatewayRequestOptions.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Dedicated Gateway Request Options
+ */
+export class DedicatedGatewayRequestOptions {
+  /**
+   * Sets the staleness value associated with the request in the Azure CosmosDB service. For requests where the {@link
```

Comment:
Please clarify its applicability on container create and also request level.

Reply:
It is in the helper method

options?: { maxIntegratedCacheStaleness: 2000 }
await db.containers.create({ ...containerDef, ...{ id } }, options);

Reply:
Am still un-clear. 

My understanding is DedicatedGateway is a service which only be created through Cosmos RP.
And the 'maxIntegratedCacheStaleness' is dedicated gateway concept only and not applicable to the core-container, so it's not applicable. Its only applicable at the request level, which hints service about cache leverage or not.

Reply:
removed